assets/xml/objects/object_kin2_obj.xml

```diff
@@ -1,25 +1,25 @@
 ﻿<Root>
     <File Name="object_kin2_obj" Segment="6">
```

Comment:
Since it looks like this whole object is now documented, can you put a description of what's in this object above the `<File>` node? There are some good examples in the codebase if you want to see what it looks like.